stock_valued_picking_report/tests/test_stock_picking_valued.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests import common
+
+
+class TestStockPickingValued(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockPickingValued, cls).setUpClass()
+        cls.tax = cls.env['account.tax'].create({
+            'name': 'TAX 15%',
+            'amount_type': 'percent',
+            'type_tax_use': 'sale',
+            'amount': 15.0,
+        })
+        cls.product = cls.env['product.product'].create({
+            'name': 'Test stuff',
+            'list_price': 100.0,
+            'tax_ids': cls.tax,
```

Comment:
Field "tax_ids" doesn't exist.

You must change this line by
```
'taxes_id': [(6, 0, [cls.tax.id])],
```

Tests doesn't fail by accident.